src/main/java/uk/gov/hmcts/befta/util/DynamicValueInjector.java

```diff
@@ -120,12 +121,15 @@ else if (anEnvVarIsStartingAt(input, pos)) {
             }
             if (jumpTo > 0) {
                 pos = jumpTo;
+                if (partValue instanceof Long || partValue instanceof Integer){
+                    outputAsNumber = partValue;
+                }
                 output.append(partValue);
             } else {
                 output.append(input.charAt(pos++));
             }
         }
-        return output.toString();
+        return outputAsNumber == null ? output.toString() : outputAsNumber;
```

Comment:
What if there are other characters in the original input and a numeric part value is just section of the output string?

Reply:
Have put a flag in to see if any part is a string, if so we return as a string which would include numbers within this